interceptors/realip/examples_test.go

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) The go-grpc-middleware Authors.
+// Licensed under the Apache License 2.0.
+
+package realip_test
+
+import (
+	"net"
+
+	"github.com/grpc-ecosystem/go-grpc-middleware/v2/interceptors/realip"
+	"google.golang.org/grpc"
+)
+
+// Simple example of a unary server initialization code.
+func ExampleUnaryServerInterceptor() {
+	// Define list of trusted peers from which we accept forwarded-for and
+	// real-ip headers.
+	trustedPeers := []net.IPNet{
+		{IP: net.IPv4(127, 0, 0, 1), Mask: net.IPv4Mask(255, 255, 255, 255)},
+	}
+	// Define headers to look for in the incoming request.
+	headers := []string{"x-forwarded-for", "x-real-ip"}
```

Comment:
What do you think about defining these header name as constants in the package, to make it easier?

Reply:
I've added them as package headers now. I originally didn't do it as to not let the package dictate what headers people should use. But I suppose making the well-known defaults available wouldn't hurt.